CHANGELOG.md

```diff
@@ -1,5 +1,37 @@
 # Changelog
 
+## [v0.18.0](https://github.com/mesg-foundation/engine/releases/tag/v0.18.0)
+
+#### Breaking Changes
+
+- ([#1536](https://github.com/mesg-foundation/engine/pull/1536)) Update process structure: rename process's key to name and move node.*.key into node.
+- ([#1540](https://github.com/mesg-foundation/engine/pull/1540)) Implement nested data map in process and change process map structure.
+- ([#1545](https://github.com/mesg-foundation/engine/pull/1545)) Add path system to reference nested data in process.
+
+#### Added
+
+- ([#1501](https://github.com/mesg-foundation/engine/pull/1501)) Add e2e process test.
+- ([#1519](https://github.com/mesg-foundation/engine/pull/1519)) Add data validation to all resources.
+- ([#1544](https://github.com/mesg-foundation/engine/pull/1544)) Add two e2e orchestrator tests.
+- ([#1546](https://github.com/mesg-foundation/engine/pull/1546)) Add e2e orchestrator behavior tests on process maps and filters.
+- ([#1547](https://github.com/mesg-foundation/engine/pull/1547)) Add e2e tests on process nested data feature.
```

Comment:
e2e shouldn't be on the added section of a changelog, added should be reserved for added features, it could also be in one single line like 'orchestrator e2e' with the list of all the PR.